### PR TITLE
feat: Change request to axios

### LINKED ISSUE #4: Use axios
I think we should use `axios` instead of `request` in this prompt. It's a library that we've used in other libraries and I think it will fit well here because of how we're using promises in most places.

### PR DESCRIPTION
**Pull request by adityavyas611**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Change request module to Axios module as per standard

**What kind of change does this PR introduce?**

- In these pull request, I have changed the request module to Axios module

**What is the new behavior?**

- The request has been sent through Axios.

Fixes: #4

### CODE REVIEW

File: lib/loadQuestions.js [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
-    );
+const loadQuestions = async formId => {
+  const body = await axios({
+    url: `https://docs.google.com/forms/d/e/${formId}/viewform`,
   });
-};
+
+  const questions = await parse(body.data);
+  return questions;
+}
```
Comment (doowb) [inline lib/loadQuestions.js]: Missing `;`

```suggestion
};
```

File: lib/loadQuestions.js [hunk 2]
```diff
@@ -1,20 +1,13 @@
...
-        if (error) {
-          reject(error);
-        } else {
-          let questions = parse(body);
-          resolve(questions);
-        }
-      }
-    );
+const loadQuestions = async formId => {
+  const body = await axios({
```
Comment (doowb) [inline lib/loadQuestions.js]: use `res` instead of `body`:

```suggestion
  const res = await axios({
```

File: lib/loadQuestions.js [hunk 3]
```diff
@@ -1,20 +1,13 @@
...
+    url: `https://docs.google.com/forms/d/e/${formId}/viewform`,
   });
-};
+
+  const questions = await parse(body.data);
```
Comment (doowb) [inline lib/loadQuestions.js]: use `res` instead of `body`:

```suggestion
  const questions = await parse(res.data);
```

**Review by doowb — CHANGES_REQUESTED** [review]
Thanks for the PR! There are a couple of things to change (linting and a variable name).

**Comment by adityavyas611** [comment]
Hi @doowb, Could you please review this and merge?

**Pull request merged**